.changeset/slimy-dragons-whisper.md

```diff
@@ -0,0 +1,7 @@
+---
+"saleor-dashboard": patch
+---
+
+Changed Product Variant details URL from `/products/<productId>/variant/<variantId>` to `/products/variant/<variantId`.
+
+Old URL will still work and redirect to new one for backwards compatibility, it will be removed in next minor release.
```

Comment:
This should be removed in Saleor 3.23 release: https://linear.app/saleor/issue/f0fad883-81ae-4cd0-82dd-98b20effa577